### ISSUE TITLE
How to add custom confetti image

### ISSUE THREAD

**Issue by flutterPratik** [funwithflutter/flutter_confetti#67]
i don't want the same stars and particles, i want to add custom png images confetti. 

like i want to add coin png to fall from the top of the screen.

Please help me if anyone knows how to add custom image in confetti effect

**Comment by amit-flutter** [comment]
Yes @flutterPratik, the same feature  I want to add custom leave image 🍁 from top to bottom. 
@shinriyo  @agent3bood  @bensonwong  @thisdarktao Could you please consider this case? I need to submit my project in a few days but I didn't find any best alternative to this.

**Comment by thisdarktao** [comment]
@amit-flutter I haven't been able to take an in-depth look, but for your use case would it be acceptable to use `createParticlePath` with a custom leaf path and provide a few autumnal colours to `colors` on the `ConfettiWidget`?

I know that isn't exactly what you want, but I'm sure @HayesGordon would appreciate a PR. While it would be possible to implement, there's going to be a tradeoff between particle count and memory use.

**Comment by HayesGordon** [comment]
Hi! Sorry for the late reply. Owing to how popular a feature request this is, it's something that I will revisit.

**Comment by techwithlanre** [comment]
> i don't want the same stars and particles, i want to add custom png images confetti.
> 
> like i want to add coin png to fall from the top of the screen.
> 
> Please help me if anyone knows how to add custom image in confetti effect

This feature is what I have been making research on for a few months now, I hope @HayesGordon and his team will be able to give us this feature, it will open up this package for more use cases

**Comment by HayesGordon** [comment]
Soon

https://media.giphy.com/media/5xtDarzgzG6eu6uVwI0/giphy.gif

**Comment by ahetawal-p** [comment]
Any new updated here ? Would love to use a custom image for our confetti.

**Comment by DarenF-20C** [comment]
seems haven't finalise it

**Comment by shinriyo** [comment]
If you like use my code.

https://github.com/funwithflutter/flutter_confetti/pull/95

**Comment by taqi92** [comment]
is there any update on this? we still can't use custom images as confetti particle
